python/mxnet/util.py

```diff
@@ -840,7 +840,7 @@ def set_np(shape=True, array=True):
 
 
 def reset_np():
-    """Deactivate NumPy shape and array semantics at the same time."""
+    """Deactivate NumPy shape and array and deafult dtype semantics at the same time."""
     set_np(shape=False, array=False)
```

Comment:
should we explicitly specify `dtype=False`?

Reply:
OK, I have added flag `dtype=False` to `set_np()`